src/components/tabs/Best.tsx

```diff
@@ -0,0 +1,258 @@
+import { type FC, useEffect, useState } from "react";
+import dayjs from "dayjs";
+import RelativeTime from "dayjs/plugin/relativeTime";
+import Skeleton from "react-loading-skeleton";
+import "react-loading-skeleton/dist/skeleton.css";
+import { toast } from "react-hot-toast";
+
+import { api } from "~/utils/api";
+import { type Record } from "@prisma/client";
+
+type BestTimerProps = {
+  timeInMs: number;
+  isRunning: boolean;
+  loading: boolean;
```

Comment:
since this is a boolean it's better to name it like this
(https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  isLoading: boolean;
```